doc/specific_iocs/dae/Experimental-Runs.md

```diff
@@ -36,9 +36,7 @@ This is from a problem see on IRIS.
 
 ## Experiment stuck in `Waiting` state after beginning a run
 
-There is now a new PV that you can write to from IBEX to force a resync of run control
-
-    caput %MYPVPREFIX%CS:RC:SYNC:SP 1
+There is now a new PV that you can write to from IBEX to [force a resync of run control](https://isiscomputinggroup.github.io/ibex_developers_manual/system_components/Run-control.html#troubleshooting)
```

Comment:
Please make this a sphinx ref so it works on shadow and gets link-checked, not an absolute url